dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -63,7 +63,7 @@ RUN opam switch create "${COMPILER}+32bit" && eval $(opam env) && \
         i386 env CC='gcc -m32' opam install zarith.1.12 && \
         opam install $BASE_OPAM
 
-# EDGE switch, dune 2.8 for OCaml 4.12, lablgtk 3.1.1 for OCaml 4.09
+# EDGE switch, dune 2.8 for OCaml 4.12, lablgtk 3.1.2 for OCaml 4.09
```

Comment:
Given that this is required by Coq anyway, just removing the comment on lablgtk would be fine as well.

Reply:
I just removed all versions in the comment, it was useless and completely out of sync anyway.